NavigationRail/src/androidTest/java/com/microsoft/device/display/samples/navigationrail/NavComponentTest.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.device.display.samples.navigationrail
+
+import androidx.compose.animation.ExperimentalAnimationApi
+import androidx.compose.foundation.ExperimentalFoundationApi
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.setValue
```

Comment:
I don't know why `ktlint` check didn't pick them up, but we don't need to import `getValue` and `setValue` here.